appengine/cloud-tasks/create_task.rb

```diff
@@ -32,15 +31,13 @@ def create_task project_id, location_id, queue_id, payload: nil, seconds: nil
   # Construct task.
   task = {
     app_engine_http_request: {
-      http_method: "POST",
+      http_method:  "POST",
       relative_uri: "/log_payload"
     }
   }
 
   # Add payload to task body.
-  if payload
-    task[:app_engine_http_request][:body] = payload
-  end
+  task[:app_engine_http_request][:body] = payload if payload
```

Comment:
Did we say we were going to drop the guard clauses?

Reply:
We did. Didn't catch all the ones I'd already changed. Thanks for pointing this out.

Reply:
BRB there's like over 100 of them :sob:

Reply:
There's another cop, Style/IfUnlessModifier, which covers this. Kinda broken as we aren't currently setting line limits. Gonna disable it.

Reply:
Style/Next also affects this